src/context/index.ts

```diff
@@ -4,9 +4,10 @@ import YAMLContext from './yaml';
 import DirectoryContext from './directory';
 
 import { isDirectory } from '../utils';
-import { version as packageVersion } from '../../package.json';
 import log from '../logger';
 
+const { version: packageVersion } = require('../../package.json')
```

Comment:
This was necessary to quell the TS compiler because the package.json file resides outside the purview of the tsconfig file.